dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -166,42 +170,38 @@ public Container getContainer(@ApiParam(hidden = true) @Auth Token authToken,
 
         return c;
     }
-    
+
     @PUT
     @Timed
     @UnitOfWork
     @Path("/{containerId}/labels")
-    @ApiOperation(value = "Update the labels linked to a container.",
```

Comment:
Is there an actual change here? Confirm with Vincent that the two of you are using the same code formatting file